processors/ttX3.py

```diff
@@ -240,7 +240,6 @@ def jetSelection(jetDict):
 analyzerChain = [
     EventSkim(selection=lambda event: event.nTrigObj > 0),
     MetFilter(
-        globalOptions=globalOptions,
```

Comment:
Any reason why the global options are omitted here?

Reply:
this is now a static variable